app/views/shop/_messages.html.haml

```diff
@@ -10,7 +10,7 @@
              register: ('<a auth="signup">' + t('.register') + '</a>').html_safe}
         - else
           = t '.require_customer_html',
-            {contact: ('<a href="##contact">' + t('.contact') + '</a>').html_safe,
+            {contact: (link_to t('.contact'), "#/contact", method: :get),
```

Comment:
I think the right solution would be a new Angular directive. In html you write something like:

```html
<a href="#contact" navigation-link>...</a>
```

And the directive is activated by the attribute `navigation-link`. It uses the `Navigation` service in `app/assets/javascripts/darkswarm/services/navigation.js.coffee` to trigger opening the contact tab.